v2/tools/generator/internal/astmodel/type_visitor_test.go

```diff
@@ -146,6 +146,8 @@ func TestIdentityVisitorReturnsEqualResult(t *testing.T) {
 
 	resource := NewResourceType(person, individual)
 
+	iface := NewInterfaceType().WithFunction(transform)
```

Comment:
Now that `NewInterfaceType()` takes functions, you might simplify this:

```suggestion
	iface := NewInterfaceType(transform)
```